### PR TITLE
Trace receiver disable debug metrics

### PR DESCRIPTION
**Pull request by basvdl**
Fixes https://github.com/grafana/k8s-monitoring-helm/issues/261

It also fixes the indentations for the `grpc` and `http` protocol blocks under the `otelcol.receiver.otlp`

### CODE REVIEW

File: charts/k8s-monitoring/templates/agent_config/_traces.river.txt [hunk 1]
```diff
@@ -6,16 +6,22 @@
 {{- if (or .Values.traces.receiver.grpc.enabled .Values.traces.receiver.http.enabled) }}
 otelcol.receiver.otlp "trace_receiver" {
 
+{{- if .Values.traces.receiver.disable_debug_metrics }}
```
Comment (skl) [inline charts/k8s-monitoring/templates/agent_config/_traces.river.txt:9]: This looks correct for the oltp receiver but it also needs to be copied over to the zipkin receiver block as well.

Comment (basvdl) [reply charts/k8s-monitoring/templates/agent_config/_traces.river.txt:9]: Gotcha.

Would we prefer to introduce 2 separate config items i.e.

```
.Values.traces.receiver. oltp .disable_debug_metrics
.Values.traces.receiver.zipkin.disable_debug_metrics
```

File: charts/k8s-monitoring/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: k8s-monitoring
 description: A Helm chart for gathering, scraping, and forwarding Kubernetes infrastructure metrics and logs to a Grafana Stack.
 type: application
-version: 0.5.1
+version: 0.5.2
```
[the remark this answers is not in the document]
Comment (basvdl) [reply charts/k8s-monitoring/Chart.yaml:5]: Cool, will revert these!

File: charts/k8s-monitoring/templates/agent_config/_traces.river.txt (hunk 1 above)
Comment (skl) [reply charts/k8s-monitoring/templates/agent_config/_traces.river.txt:9]: @basvdl that's probably the most robust option I guess, to have individual disable per receiver. Go for it.

**Review by skl — APPROVED** [review]
💯

**Pull request merged**